GTViewController/Classes/Custom Classes/GTFollowupModalView.m

```diff
@@ -115,7 +97,27 @@ - (instancetype)initFromElement:(TBXMLElement *)element withStyle:(GTPageStyle*)
                 currentY += buttonPairView.frame.size.height + 10;
                 
             }else if ([modalComponentElementName isEqual:kName_Thank_You]) {
-                self.thankYouPage = nil;
+                
```

Comment:
This method is very large. You might want to consider splitting these if statements into their own methods